openIt/ngsild-payloads/authorization.jsonld

```diff
@@ -78,5 +78,19 @@
   "generatedUntil": {
     "type": "Property",
     "value": "2024-09-02T23:59:59.000Z"
-  }
+  },
+  "viewedBy": [
+    {
+      "type": "Relationship",
+      "object": "urn:ngsi-ld:UserProfile:123",
+      "datasetId": "urn:ngsi-ld:Dataset:2024-11-25-09-01-00-123",
+      "observedAt": "2024-11-25T09:01:00Z"
+    },
+    {
+      "type": "Relationship",
+      "object": "urn:ngsi-ld:UserProfile:123",
+      "datasetId": "urn:ngsi-ld:Dataset:2024-11-24-09-01-00-123",
+      "observedAt": "2024-11-24T09:01:00Z"
+    }
+  ]
```

Comment:
don't think you need different instances / datasetIds here (even more with the same object). it can be the same instance and you update object and observedAt each time a user sees the authorization.

Reply:
Understood, will make the changes